### PR TITLE
Update S3 docs

### PR DESCRIPTION
**Pull request by sudoyu**
We are tried to use S3 Writer, and found the S3 docs are outdated.
I read moloch source codes and updated proper features of S3 Writer.
I hope it helps.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: _wiki/s3.md [hunk 1]
```diff
@@ -65,4 +95,7 @@ s3SecretAccessKey=THESECRETACCESSKEY_FOR_MOLOCHVIEWER
 s3ExpireDays=10
 ```
 
+**WARNING** - After Moloch 2.1.0, accesskey-based authentication is not recommended. It should be obtained from EC2 IAM Role.
```
Comment (awick) [inline _wiki/s3.md]: Why is it not recommended?

File: _wiki/s3.md [hunk 2]
```diff
@@ -10,19 +10,27 @@ permalink: "/s3"
...
 
-**WARNING** - Currently all the writing from moloch-capture to S3 is NOT sent over TLS, is NOT compressed, and is NOT AMZ server sided encrypted at rest. Waiting to get some feedback or see how we use this before adding a bunch more features.
+Currently all the writing from moloch-capture to S3 is sent over TLS(HTTPS S3 REST endpoint). To make these PCAPs with server side encrypted, consider to configure S3 bucket's [default encryption settings](https://docs.aws.amazon.com/AmazonS3/latest/dev/bucket-encryption.html).
+
+Sending with compression, storing gzipped files, max connections, and max requests are configurable options. See [Writer S3 Settings](https://molo.ch/settings#writer-s3-settings).
+
+**WARNING** - In Moloch 2.0.1, all reading by moloch-viewer from S3, is NOT work properly due to [S3 plugin issue](https://github.com/aol/moloch/issues/1178).
```
Comment (awick) [inline _wiki/s3.md]: I think this was both 2.0 and 2.0.1 that are broken.  But I think lets just remove this line.

File: _wiki/s3.md (hunk 1 above)
Comment (sudoyu) [reply _wiki/s3.md]: Thanks pointed out!
Using EC2 Role is better practice on AWS. `not recommended` is not a suitable sentence. I will rewrite it.

File: _wiki/s3.md (hunk 2 above)
Comment (sudoyu) [reply _wiki/s3.md]: I will do that. thanks!

**Comment by sudoyu** [comment]
I will add some explanation about **maxFileSizeG** this weekend.

File: _wiki/s3.md (hunk 1 above)
Comment (pjsg) [reply _wiki/s3.md]: If the capture process is not running in AWS, then there is no choice but to use access keys...

Comment (sudoyu) [reply _wiki/s3.md]: Right!
Sorry to lack of consideration in case from on-premises to S3.

**Comment by sudoyu** [comment]
@awick @pjsg thanks for reviews!
Also I've updated about some configurations(maxFileSizeG, maxFileTimeM, -d option).

**Pull request merged**